general/solomiia_tymoshchuk/Task6.java

```diff
@@ -0,0 +1,61 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.HashMap;
+import java.util.Objects;
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts two numbers and a symbol of an operation
+ * (as a character) and outputs the result of the selected operation.
+ * Possible characters for the operation: ‘+’, ‘-’, ‘*’, ‘/’, ‘%’.
+ * The program should work only with integer values.
+ * Create two versions of the program: one should use the if-else statement, second - the switch statement.
+ **/
+public class Task6 {
+    HashMap<Integer, String> weekDays = new HashMap<Integer, String>();
+    Scanner sc = new Scanner(System.in);
+    double number = sc.nextDouble();
+    double number1 = sc.nextDouble();
```

Comment:
instead of storing values as instance variables, it'd be better to pass them as method arguments - that would allow to use same methods with different values (in case, we'd like to build such program).